src/frontend/komponenter/Fagsak/OpprettBehandling/OpprettBehandlingProvider.tsx

```diff
@@ -105,8 +112,12 @@ const opprettBehandlingReducer = (state: IState, action: IAction): IState => {
     }
 };
 
-const OpprettBehandlingProvider: React.StatelessComponent = ({ children }) => {
-    const [state, dispatch] = React.useReducer(opprettBehandlingReducer, initialState);
+interface IProps {
+    fagsak: IFagsak;
+}
+
+const OpprettBehandlingProvider: React.StatelessComponent<IProps> = ({ fagsak, children }) => {
```

Comment:
Dette vet nok du mer om enn meg, men Intellij klager på at StatelessComponent/SFC er deprecated :/ 

Ikke stateless siden 16.8 står det her: https://stackoverflow.com/questions/54706996/difference-between-react-functioncomponent-and-react-sfc

Reply:
(foreslår FunctionComponent)

Reply:
Fikser 👍